src/pveclib/vec_f128_ppc.h

```diff
@@ -4957,7 +6260,287 @@ static inline vec_xscvudqp (vui64_t int64)
       result = vec_xsiexpqp (q_sig, q_exp);
     }
 #else
-  result = f64[VEC_DW_H];
+  result = int64[VEC_DW_H];
+#endif
+  return result;
+}
+
+/** \brief VXS Scalar Convert Signed-Quadword to Quad-Precision format.
+ *
+ *  The signed quadword element of vector int128 is converted
+ *  to quad-precision format.
+ *  If the conversion is not exact the default rounding mode is
+ *  "round to Nearest, ties to even".
```

Comment:
s/"round to Nearest, ties to even"/Round to Even/ ?

Reply:
"Round to Nearest Even"